.eslintrc

```diff
@@ -35,6 +35,8 @@
     "indent": ["error", 2]
   },
   "env": {
+    "es6": true,
+    "jest": true,
```

Comment:
for the eslint config we should just use https://github.com/babel/eslint-config-babel. I would do this in a new pr after this is merged so its easier to review